setup.py

```diff
@@ -23,7 +23,7 @@ def version():
 
 
 def load_deps(file_name):
-    """Load dependencies from requirements file"""
```

Comment:
Why? These changes makes it hard to focus on the real changes. Also `"""` is [the convention](https://www.python.org/dev/peps/pep-0257/).

Reply:
Updated.